hv-rhel7.x/hv/netvsc.c

```diff
@@ -896,11 +892,17 @@ int netvsc_send(struct net_device_context *ndev_ctx,
 			msd_len = 0;
 		}
 	}
+	/* Keep aggregating only if stack says more data is coming
+	 * and not doing mixed modes send and not flow blocked
+	 */
+	xmit_more = skb->xmit_more &&
```

Comment:
Can this line compile on RHEL 7.0 and 7.1, which may not have the field skb->xmit_more? I didn't check the 7.0/7.1 kernel code, but I think the old LIS-next code in netvsc_copy_to_send_buf() has the conditional compilation, which implies the field skb->xmit_more doesn't exist.

Reply:
Oh, please ignore this comment. I saw  a later patch fixed this. Can you merge the 2 patches into one? :-)